compiler/Cargo.toml

```diff
@@ -1,26 +1,21 @@
 [package]
-name = "contract"
-version = "3.0.0-rc9"
+name = "flipper"
+version = "0.1.0"
+authors = ["[your_name] <[your_email]>"]
 edition = "2021"
-authors = ["Achim Schneider <achim@parity.io>"]
-license = "Apache-2.0"
-readme = "README.md"
 
 [dependencies]
-ink_primitives = { version = "3.0.0-rc9", default-features = false }
-ink_metadata = { version = "3.0.0-rc9", default-features = false, features = ["derive"], optional = true }
-ink_env = { version = "3.0.0-rc9", default-features = false, features = ["ink-debug"] }
-ink_storage = { version = "3.0.0-rc9", default-features = false }
-ink_lang = { version = "3.0.0-rc9", default-features = false }
-ink_prelude = { version = "3.0.0-rc9", default-features = false }
-ink_eth_compatibility = { version = "3.0.0-rc9", default-features = false }
+ink_primitives = { version = "3.0", default-features = false }
```

Comment:
Can you split out the ink! 3.0 update into its own PR?

Reply:
True, upgrading to ink! 3.0 is the only change not related to the test feature. The reason is that the Dockerfile stopped building with ink! "3.0.0-rc9" since it was pulling the latest version of carg-contract fro Git, which has been updated. I can try to revert back to "3.0.0-rc9".